lib/googleDrive/googleAuth.js

```diff
@@ -0,0 +1,15 @@
+import GoogleAuth from 'google-auth-library';
+
+export default function (googleApi) {
```

Comment:
avoid anonymous functions, give it a name. `default export` should have the same name as a file name